src/routes/nominationRouter.js

```diff
@@ -1,5 +1,5 @@
 import _ from 'lodash';
```

Comment:
This router is still in the old format. if you don't want this router, remove that.

Reply:
will be updated later.